raysort/sort_utils.py

```diff
@@ -261,6 +270,7 @@ def _run_valsort(argstr: str):
     if proc.returncode != 0:
         logging.critical("\n%s", proc.stderr.decode("ascii"))
         raise RuntimeError(f"Validation failed: {argstr}")
+    return proc.stderr
```

Comment:
Can't comment on the method definition line, but since we're returning this, add a return type to be `str`? So something like
```
def _run_valsort(argstr: str) -> str:
```

Reply:
+1